.github/pull_request_template.md

```diff
@@ -1,11 +1,13 @@
 **Please make sure to target feature and bugfix requests to develop branch**
```

Comment:
Please revert those changes. This is the PR template and should remain **generic**. It will automatically be used on every new PR and can be customized there! ;)